### PR TITLE
docs: update project status

### PR DESCRIPTION
**Pull request by killme2008**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## What's changed and what's your intention?

Update project status to encourage developers to try it.

## Checklist

- [ ] I have written the necessary rustdoc comments.
- [ ] I have added the necessary unit tests and integration tests.
- [ ] This PR requires documentation updates.

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -154,6 +154,8 @@ The current version has not yet reached General Availability version standards.
 In line with our Greptime 2024 Roadmap, we plan to achieve a production-level
 version with the update to v1.0 in August. [[Join Force]](https://github.com/GreptimeTeam/greptimedb/issues/3412)
```
Comment (tisonkun) [inline README.md:155]: 1.0 is now planned to be releases at the end of this year?

Comment (killme2008) [reply README.md:155]: Yes, need to be updated

Comment (tisonkun) [reply README.md:155]: Maybe simply delete this paragraph

Comment (killme2008) [reply README.md]: Updated the description.

**Review by tisonkun — APPROVED** [review]

**Review by sunng87 — APPROVED** [review]

**Pull request merged**